figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -98,6 +109,12 @@ for (dataset in c("tcga", "pbta")) {
       as.data.frame() %>%
       rowid_to_column("index")
 
+    # Export
+    all_caller_df %>%
+      dplyr::select(Kids_First_Biospecimen_ID = Tumor_Sample_Barcode, everything()) %>%
+      dplyr::arrange(Kids_First_Biospecimen_ID) %>%
+      readr::write_csv(figS2abc_csv)
```

Comment:
Do we need `everything()` here? Never mind... keeping the position & variant info seems worthwhile to me. I would drop `index` though, as noted.